src/wx/panel.cpp

```diff
@@ -1658,6 +1658,8 @@ class FilterThread : public wxThread {
             done->Post();
             continue;
         }
+
+	return 0;
```

Comment:
It needed a default return value here, in case the `while` loop never ends.

Is `return 0` a valid default? cc @rkitover @ZachBacon